packages/components/src/FigspecViewer/ViewerMixin.ts

```diff
@@ -97,6 +99,11 @@ export const ViewerMixin = <T extends Constructor<LitElement>>(
             z-index: var(--z-index);
           }
 
+          .spec-canvas-wrapper {
+            width: 100vw;
+            height: 100vh;
```

Comment:
```suggestion
            position: absolute;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
```

As being CustomElement, it should respect the size of `:host`. (e.g. `figspec-frame-viewer { width: 400px; height: 200px; }`)

Reply:
I see, makes sense. I tried this before using vw/vh but it didn't work:
![image](https://user-images.githubusercontent.com/1671563/103938304-531fc780-512a-11eb-971c-ccc158dcc73d.png)